jiup/rules/rules.go

```diff
@@ -577,11 +577,10 @@ func init() {
 	Rule("geforce-experience",
 		v.Regexp(
 			"https://www.nvidia.com/en-us/geforce/geforce-experience/",
-			h.Re("https://us.download.nvidia.com/GFE/GFEClient/([0-9.]+)/GeForce_Experience"),
+			h.Re("https://.*.download.nvidia.com/GFE/GFEClient/([0-9.]+)/GeForce_Experience"),
 		),
-		d.HTMLA(
-			"https://www.nvidia.com/en-us/geforce/geforce-experience/",
-			"a[href^='https://us.download.nvidia.com/GFE/GFEClient/'][href$='.exe']",
+		d.Template(
+			"https://us.download.nvidia.com/GFE/GFEClient/{{.Version}}/GeForce_Experience_v{{.Version}}.exe",
```

Comment:
If you needed to change the version extractor's subdomain to a wildcard, I'd rather that you use a dynamic download extractor as well.

Reply:
@geek1011 Why? The subdomain in the link extracted dynamically will depend on where the server on which the updater is run, what makes even less sense.

Reply:
Mainly because it works better with the link checking in the updater and is easier to manage when forcing an update.

Note that the updater is run in the US.

Reply:
I'd suggest changing both extractors to use the HTML link one with the class `btn-download-manual`.